amazonredshift_blueprints/upload_file.py

```diff
@@ -235,12 +196,16 @@ def main():
     if schema == '':
         schema = None
 
-    db_string = create_connection_url(host = host, password = password, user = user, database = database)
+    if port in ('', None):
+        db_string = create_connection_url(host=host, password=password, user=user, database=database)
```

Comment:
Decided to go this route so that the default value for the create connection function could be use if port comes in blank. Port wasn't being used in the script prior